docs/howto/build-run.md

```diff
@@ -268,6 +280,16 @@ If you've run `react-native link`, you can discard the edits it made
 (along with any other edits you've made) by running `git reset --hard`.
 
 
+### Build failure: missing `/node_modules/react-native/third-party/double-conversion-1.1.6/src/fast-dtoa.cc`
+
+When trying to build the iOS app, if you get an error like
+
+```
+error: Build input file cannot be found: '/Users/chrisbobbe/dev/zulip-mobile/node_modules/react-native/third-party/double-conversion-1.1.6/src/bignum.cc' (in target 'double-conversion' from project 'React')
```

Comment:
Unsure about having my name in here (I don't really mind); removing it might lose clarity.

Reply:
Apple occasionally uses `jappleseed` for this purpose. ([Example.](https://developer.apple.com/documentation/foundation/filemanager/1407693-url#discussion))

Reply:
Done 👍